scripts/startup/start_panoptes_in_tmux.sh

```diff
@@ -97,7 +97,13 @@ set -x
 
 # Create a window running the zeromq message forwarders.
 # These provide connections between message publishers and subscribers.
-create_and_init_window messaging start_messaging_hub.sh
+create_and_init_window msg_hub start_messaging_hub.sh
+
+# Create a window running the social messaging program. It subscribes to
+# zeromq and posts some messages to Twitter or Slack, if
```

Comment:
nitpick: could expand beyond twitter or slack

Reply:
Agreed, wasn't sure what to say for a more general description... after all, it could post to any site, not just a social messaging service.